test/test_xpath_parser.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: false
+#
+#  Created by Henrik Mårtensson on 2007-02-18.
+#  Copyright (c) 2007. All rights reserved.
```

Comment:
You didn't write these tests, right?

Reply:
If you don't object, we want to use "Copyright (C) 1993-2013 Yukihiro Matsumoto" mentioned in https://github.com/ruby/rexml/blob/master/LICENSE.txt .

If we don't mention copyright explicitly here, "Copyright (C) 1993-2013 Yukihiro Matsumoto" is used.

Reply:
Removed

Reply:
Thanks.

Note: This assumes that copied code (copyrighted code in an existing test) will be removed finally.